plugins/nsmserver/nsmserver_client.go

```diff
@@ -322,13 +418,15 @@ func startClientServer(id string, endpoints *nsmClientEndpoints) {
 	}
 
 	unix.Umask(socketMask)
-	sock, err := net.Listen("unix", listenEndpoint)
+
+	sock, err := newCustomListener(listenEndpoint) // net.Listen("unix", listenEndpoint)
 	if err != nil {
 		logger.Errorf("failure to listen on socket %s with error: %+v", client.socketPath, err)
 		client.allocated = false
 		return
 	}
-	grpcServer := grpc.NewServer([]grpc.ServerOption{}...)
+
+	grpcServer := grpc.NewServer( /*grpc.UnaryInterceptor(grpcUnaryIntercept)*/ )
```

Comment:
What is this comment?

Reply:
I was fighting with grpc, so it is a leftover of that fight. will remove.